packages/perseus/src/widgets/numeric-input.jsx

```diff
@@ -547,10 +554,10 @@ type RenderProps = {|
 // can accept several input forms with or without "%", the decision
 // to parse based on the presence of "%" in the input, is so that we
 // don't accidently scale the user typed value before grading, CP-930.
-export const maybeParsePercentInput: ($FlowFixMe, $FlowFixMe) => $FlowFixMe = (
-    inputValue,
-    normalizedAnswerExpected,
-) => {
+export const maybeParsePercentInput: (
+    string | number,
+    boolean,
+) => string | number = (inputValue, normalizedAnswerExpected) => {
```

Comment:
I've been re-thinking my tendency to make everything an arrow function. They are extremely handy when you need to ensure correct `this` binding, but in this case, I wonder if this suggestion is clearer and easier to read. 

In short, do we _need_ an arrow function here?

```suggestion
export function maybeParsePercentInput: (
    inputValue: string | number,
    normalizedAnswerExpected: boolean,
): string | number {
```

Reply:
Are there other benefits to not using an arrow function other than ease of reading? Just curious as we basically learned to use arrow functions all the time.

Reply:
Historically, I've been very pro-arrow functions. But in some cases `function` is cleaner (like this case). This question has been asked a few times:
* https://khanacademy.slack.com/archives/C02NMCKHD/p1664846299539919
* https://khanacademy.slack.com/archives/CEA6W0F6F/p1652802419917699

Reply:
@Myranae I think JS devs were just really excited to get them and for awhile used them for everything. It was kind of a cool thing for awhile; I remember a time when every tech talk had a 2 minute aside to explain what an arrow function was. It makes a lot of sense for callback arguments:

``` JS
const numbers = [1, 2, 3]

// olden JS would have looked something like
numbers.map(function (num) { return num * num })

// new JS would look something like
numbers.map(num => num * num)
```

I actually prefer `function funName() {}` myself when it's a named function; I just think it's easier to parse. The big differences are:

- A different `this`. `function` creates a new `this` context and to get around that devs had to use things like `bind`. Not as big of a deal today since the JS community got behind functional programming with pure functions.
- One liner arrow functions can return without the `return` keyword.